### ISSUE TITLE
Any Plans for a Gulp plugin

### ISSUE THREAD

**Issue by jameseisenhauer** [webdriverio/webdriverio#418]
Noticed you have grunt plugins with the project.  Was wondering if there are any plans for gulp plugin.

thanks

**Comment by christian-bromann** [comment]
A gulp plugin would be pretty nice to have. I created a new repository/project our webdriverio origanisation called [gulp-webdriver](https://github.com/webdriverio/gulp-webdriver). Feel free to start working on it if you want. Hope I get time soon to make this available. I guess after I've published WebdriverIO v2.5.0 (has multibrowser functionality) I will enhance `grunt-webdriver` to support multiple browser per test in parallel and at the same time I will put effort in developing gulp-webdriver.

Closing issue, everything related to gulp-webdriver has an own place now

**Comment by binarykitchen** [comment]
Great, looking forward to it!
